static/css/components.css

```
@@ -250,6 +250,9 @@ button:disabled {
 .group-info {
     flex: 1 1 200px;
     min-width: 0;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)